regions/shapes/annulus.py

```diff
@@ -197,7 +257,7 @@ def to_sky_args(self, wcs):
 
 
 @six.add_metaclass(abc.ABCMeta)
-class AsymmetricAnnulusSkyRegion(CompoundSkyRegion):
+class AsymmetricAnnulusSkyRegion(SkyRegion):
     """
     A rectangular annulus in `~astropy.coordinates.SkyCoord` coordinates.
```

Comment:
@cdeil I think it was this that threw me off; I didn't see the `RectangleAnnulus*` regions below that exist, which are totally fine.  This line should probably be
```suggestion
    An asymmetric annulus in `~astropy.coordinates.SkyCoord` coordinates; this is the base class for such region types (e.g., Rectangular and Elliptical).
```
or similar

Reply:
(again, I realize that's beyond what's in this PR... this does not necessarily need to be changed here, it just caught my eye when I was skimming)